src/hotspot/share/utilities/vmError.cpp

```diff
@@ -422,6 +468,24 @@ static void print_oom_reasons(outputStream* st) {
   st->print_cr("# This output file may be truncated or incomplete.");
 }
 
+static void print_stack_location(outputStream* st, void* context, int& continuation) {
+  int i = continuation;
+  // Update continuation with next index before fetching frame
+  continuation = i + 1;
+  const frame fr = os::fetch_frame_from_context(context);
+  while (i < 8) {
```

Comment:
Can we name this constant (function scope const is fine, something like "number_of_stack_slots" or so).

Reply:
Done